src/Beman/Optional26/tests/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# cmake-format: off
+# src/Beman/Optional26/tests/CMakeLists.txt -*-makefile-*-
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# cmake-format: on
+
+# Tests
+add_executable(optional_test "")
+
+target_sources(
+  optional_test
+  PRIVATE optional.t.cpp optional_ref.t.cpp
+          optional_monadic.t.cpp optional_range_support.t.cpp
+          optional_ref_monadic.t.cpp detail/iterator.t.cpp)
+
+target_link_libraries(optional_test "${TARGET_LIBRARY}" GTest::gtest GTest::gtest_main)
```

Comment:
Did you consider using `beman_optional26` instead of  using "${TARGET_LIBRARY}"? I've seen this pattern used in CMake code, but it would be more readable to use the variable with a good name (`beman_optional26`) instead of something generic and ambiguous like TARGET_LIBRARY.